### DISCUSSION TITLE
How do i login when verification email is deleted?

### DISCUSSION THREAD

**Discussion by I-Margarini** [community/community#140508] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

My current email used for verification on the account has been deleted by the administrator. Every time I log in it says that have to verify and sends the verification code to the old email. Is there a work around I can use to not lose my stuff. I pay for copilot and I don't even think I can turn it off unless I tell the bank to block the payment.

**Comment by Topromm** [comment]
Try reaching out to GitHub Support directly so they can help update your email on file. They should be able to verify your identity and assist with getting your account linked to a new email. Getting a reply might take a while as they're getting a lot of new tickets, so you might want to consider blocking the payment if it's taking a long time to get in contact.

**Reply by I-Margarini** [reply]
What is the email address? I am trying to but cant login.

**Reply by Topromm** [reply]
@I-Margarini You can reach out [here](https://support.github.com/request) by choosing `2FA Issues` or `open a new ticket` at the top. Just let them know your issue is about a different account.

**Reply by I-Margarini** [reply]
![image](https://github.com/user-attachments/assets/8641dd76-a2a3-4732-929e-0e1ec139f86a)

This is what the robot is giving me

**Reply by Topromm** [reply, marked as answer]
Try lowercase, if the bot isn't helpful it might be easier to contact support through a ticket.

**Reply by I-Margarini** [reply]
Yea it needed lower case. Thanks I sent a ticket.

**Comment by ebndev** [comment]
Hi @Topromm, thanks for taking the time to answer!

Hi @I-Margarini, you have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact). As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. 

We appreciate your understanding!
